src/App.js

```diff
@@ -1,23 +1,30 @@
+// src/App.jsx
 import React from 'react';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './App.css';
 import Home from './Screens/Home/Home';
 import TicketDetails from './Screens/TicketDetails/TicketDetails';
 import Sidebar from './Components/Sidebar/Sidebar';
+import { ErrorProvider } from './contexts/ErrorContext';
+import ErrorModal from './Components/ErrorModal/ErrorModal';
```

Comment:
No changes should be present in `App.js` please remove once the code is tested

Reply:
ok